app/assets/stylesheets/logixRelease.scss

```diff
@@ -1,3 +1,7 @@
+body {
+  padding-top: 90px;
```

Comment:
Curious Why ?

Reply:
So that the top content doesn't overlap with the navbar